tsl/src/hypercore/utils.c

```diff
@@ -38,7 +38,7 @@
  * method back to "heap".
  */
```

Comment:
It'd be nice to update the comment here as this function now can both turn on and off autovacuum.